apps/studio.giselles.ai/emails/README.md

```
@@ -51,6 +52,235 @@ pnpm -F studio.giselles.ai email:export
 
+## Marketing Emails
+
+Marketing emails are stored in the `marketing/` directory. These templates are used for promotional and marketing campaigns.
+
+### 📘 Giselle Email Style Guide v1
+
+#### 🌐 Overview
+
+Giselle marketing emails are designed with the purpose of:
+
+**"Brand experience as an extension of product experience"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)